crates/astria-proto/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+pub mod execution {
```

Comment:
Should we name the module matching the protobuf package name? Right now the `v1` bit gets lots.

That would be `pub mod astria_execution_v1`, which looks awully verbose (think `use astria_proto::astria_execution_v1::SomeType`)

Maybe nested modules? E.g.

```rust
pub mod execution {
    pub mod v1 {
        include!(...)
    }
}
```

Reply:
I like the idea of nested modules here, but I am not sure that I love the idea of duplicating `astria`. 

What if crates were:
`astria_proto::execution::v1`

The difference between package name and module is then that in protobuf we don't specify that it's proto, because obviously.